components/DuplicateWarningPanel/DuplicateWarningPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import cx from 'classnames';
+import s from './DuplicateWarningPanel.module.scss';
+import { LegacyResident } from 'types';
+import axios from 'axios';
+import ResidentsTable from 'components/Search/results/ResidentsTable';
+
+interface Props {
+  newResident: Record<string, unknown>;
+}
+
+const DuplicateWarningPanel = ({
+  newResident,
+}: Props): React.ReactElement | null => {
+  const [matchingResidents, setMatchingResidents] = useState<LegacyResident[]>(
+    []
+  );
+
+  useEffect(() => {
+    axios
```

Comment:
I think this should use the `useResidents()` hook instead of re-implementing the axios / networking behaviour inside this component?

Reply:
i avoided it because it looked excessive for what we needed (infinite loading, etc) but maybe it's fine

Reply:
Sure – I think better to reuse or expand the shared capability given we have it, rather than reimplement

Reply:
done